README.rst

```diff
@@ -86,6 +86,8 @@ Simple consumer:
 
 
     async def main(loop):
+        # Connect with the givien parameters is also valiable.
+        # aio_pika.connect_robust(host="host", login="login", password="password")
```

Comment:
I think it's important to mention that this is mutually exclusive.

Reply:
@mosquito  What's that mean ? sorry I am new in contributor

Reply:
You can pass `url` or `kw`-based params not both.

Reply:
Yes of course